nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -271,20 +305,44 @@ define([
              * generate the parameter ids, and make a structure like:
              * {
              *   fileType1: {
-             *      param1: '' or default from spec,
-             *      param2: '' or default from spec,
-             *      ...etc.
-             *   },
+             *      params: {
+             *          param1: '' or default from spec,
+             *          param2: '' or default from spec,
+             *          ...etc.
+             *      },
+             *      filePaths: [{
+             *
+             *      }],
              *   fileType2: {...}
              * }
              *
              */
-            const initialParams = {};
+            const initialParams = {},
+                fileParamIds = {},
+                otherParamIds = {};
+            /* initialize the parameters set.
+             * Get the app spec and split the set of parameters into filePaths and params
+             * each input file (typesToFiles[fileType].files) gets its own
```

Comment:
gets its own... what? Don't leave us in suspense!

Reply:
Oops. Removed suspense...